drivers/counter/counter_ll_stm32_timer.c

```diff
@@ -53,6 +54,12 @@ static uint32_t(*const get_timer_compare[TIMER_MAX_CH])(TIM_TypeDef *) = {
 	LL_TIM_OC_GetCompareCH3, LL_TIM_OC_GetCompareCH4,
 };
 #endif
+#ifdef CONFIG_COUNTER_CAPTURE
+static uint32_t (*const get_timer_capture[TIMER_MAX_CH])(const TIM_TypeDef *) = {
```

Comment:
```c
/** Channel to timestamp capture function mapping. */
```
Could you add an empty line before `#ifdef ...` above and after `#endif` below?

Reply:
done